### ISSUE TITLE
Title colour above Map graph

### ISSUE THREAD

**Issue by gcarreno** [wp-xyz/corona#36]
Hi Werner(@wp-xyz),

It may be my bad eye sight, I have a very crappy varifocal set of glasses and they need to be replaced, but does this colour look good to you on dark mode:

![corona-map-title-bad-colour](https://user-images.githubusercontent.com/81799/108931045-e65f7e80-763e-11eb-8e00-0bbdfedf6cc7.png)

Don't worry about the cropping, I did size the window really small to make the screenshot image small.

Cheers,
Gus

**Comment by wp-xyz** [comment]
Switched my LMDE 4 Cinnamon to dark mode and can confirm the issue. Additionally, it is not correct that, in dark mode, the chart background is white  and that the axis lines are black. I changed the Title.Color to clWindowText, the Title.Brush.Style to bsClear,the background colors to clWindow, the axis lines and the frame line to clWindow and the grid line to a color derived from the chart background color. This seems to be correct.

Ideally clDefault should do this correctly. The problem with TAChart is, however, that it can paint using a variety of backends which not necessarily can resolve clDefault to a meaningful color. This probably is the reason why so many colors are hard-coded to fixed colors.

**Comment by gcarreno** [comment]
Hey Werner(@wp-xyz),

Thank you so much for the better colour selection! Had a drive on latest `master` and `mapping` and it's so pleasing to the eye!!

> Ideally `clDefault` should do this correctly.

Yeap I agree and since I've been using Dark Mode on Ubuntu I've also identified a ton of crappy colour combinations.
Not sure if it's the Ubuntu colour scheme or the way Lazarus maps those colours to the non specific colour names, like `clDefault`, `clWindowstext`, etc...

Cheers,
Gus

**Comment by wp-xyz** [comment]
I just updated TAChart so that it makes all relevant pen colors default to clDefault. The problem was that the chart pens did not resolve clDefault and usually become black, in particular on other drawing backends. But now I found a TControl method, GetDefaultColor, which determines the correct rgb value for a brush and for a font when their color is set to clDefault, and I assumed that a pen should have the same default color as a font. There are some strange things going on with clDefault brush colors, so I decided to default Chart.Color to clForm and Chart.BackColor to clWindow. But in total, I think the default charts look quite nice now in both normal and dark mode.
